src/main/java/de/zalando/zmon/metriccache/HttpClientConfig.java

```diff
@@ -0,0 +1,54 @@
+package de.zalando.zmon.metriccache;
+
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.config.RequestConfig;
+import org.apache.http.client.fluent.Executor;
+import org.apache.http.impl.client.HttpClients;
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Scope;
+
+import java.util.concurrent.TimeUnit;
+
+@Configuration
+@ConfigurationProperties(prefix = "zmon.metricsconfig")
+public class HttpClientConfig {
+
+    private final int connectTimeout = 3000;
+    private final int socketTimeout = 30000;
+    private final int connTimeToLive = 120_000;
```

Comment:
is this intentional --> 120_000

Reply:
yes - it's to make it easier to read
> In Java SE 7 and later, any number of underscore characters (_) can appear anywhere between digits in a numerical literal. This feature enables you, for example, to separate groups of digits in numeric literals, which can improve the readability of your code.

© https://docs.oracle.com/javase/7/docs/technotes/guides/language/underscores-literals.html